crates/vm/src/system/memory/offline.rs

```diff
@@ -56,12 +60,14 @@ impl<F: PrimeField32> OfflineMemory<F> {
         memory_bus: MemoryBus,
         range_checker: SharedVariableRangeCheckerChip,
         timestamp_max_bits: usize,
+        config: MemoryConfig,
     ) -> Self {
```

Comment:
note: We should probably stop passing `initial_memory` here since I think it's only used with `set_initial_memory`.